pyscriptjs/src/plugin.ts

```diff
@@ -60,6 +60,26 @@ export class Plugin {
      */
     afterPyScriptExec(options: {interpreter: Interpreter, src: string, pyScriptTag: PyScriptTag, result: any}) {}
 
+    /** The source of the <py-repl> tag has been fetched and its output-element determined;
+     * we're about to evaluate the source using the provided runtime
+     *
+     * @param options.interpreter The Runtime object that will be used to evaluated the Python source code
+     * @param options.src {string} The Python source code to be evaluated
+     * @param options.outEl The element that the result of the REPL evaluation will be output to.
+     * @param options.pyReplTag The <py-repl> HTML tag the originated the evaluation
+     */
+    beforePyReplExec(options: {interpreter: Interpreter, src: string, outEl: HTMLElement, pyReplTag: any}){}
+
+    /**
+     *
+     * @param options.interpreter  The Runtime object that will be used to evaluated the Python source code
```

Comment:
Adding the suggestion if you think it should be interpreter 😄 
```suggestion
     * @param options.interpreter The Runtime object that will be used to evaluate the Python source code
```